From c5ec50a9905af0d8333309417d0f79ca94351b92 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 12 May 2022 14:31:43 +0100
Subject: [PATCH 430/546] media: i2c: Update ov9281 Kconfig entry

Bring the OV9281 Kconfig declaration in line with upstream entries.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/media/i2c/Kconfig | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/media/i2c/Kconfig
+++ b/drivers/media/i2c/Kconfig
@@ -1379,7 +1379,6 @@ config VIDEO_TW2804
 config VIDEO_OV9281
 	tristate "OmniVision OV9281 sensor support"
 	depends on I2C && VIDEO_DEV
-	depends on MEDIA_CAMERA_SUPPORT
 	help
 	  This is a Video4Linux2 sensor-level driver for the OmniVision
 	  OV9281 camera.
